subsys/partition_manager/pm.yml.tfm

```diff
@@ -19,7 +19,7 @@ tfm_secure:
 tfm_nonsecure:
   span: [app]
 
-#ifdef CONFIG_TFM_STORAGE
+#ifdef CONFIG_PM_PARTITION_TFM_STORAGE
```

Comment:
maybe grep for CONFIG_TFM_STORAGE and check if there are any SYSBUILD scripts using it, assuming we are changing it's semantics.

Reply:
I haven't seen anything at all regarding sysbuild and TF-M flash/ram layout.
Our out-of-tree support still rely only on PM support.
@nordicjm Any input here?

Reply:
Sysbuild uses the existing cmake code for TF-M, you could try the PSA sample building using sysbuild (if it uses this) to check it is still working, or any other sample that does, and ensure it builds and runs properly and that the partitions.yml file has the same sizes and offsets as the child/parent image build